Anytype/Sources/PresentationLayer/Flows/HomeFlow/HomeCoordinatorViewModel.swift

```diff
@@ -310,6 +311,10 @@ final class HomeCoordinatorViewModel: ObservableObject,
             try await push(data: editorData)
         case .spaceShareTip:
             showSpaceShareTip = true
+        case .membership(let tierId):
+            keyboardToggle.toggle()
```

Comment:
Not needed `handleAppAction` contains `keyboardToggle.toggle()` and `dismissAllPresented `